Remotion/Web/Core/ExecutionEngine/WxeUrlSettings.cs

```diff
@@ -0,0 +1,83 @@
+﻿// This file is part of the re-motion Core Framework (www.re-motion.org)
+// Copyright (c) rubicon IT GmbH, www.rubicon.eu
+//
+// The re-motion Core Framework is free software; you can redistribute it
+// and/or modify it under the terms of the GNU Lesser General Public License
+// as published by the Free Software Foundation; either version 2.1 of the
+// License, or (at your option) any later version.
+//
+// re-motion is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU Lesser General Public License for more details.
+//
+// You should have received a copy of the GNU Lesser General Public License
+// along with re-motion; if not, see http://www.gnu.org/licenses.
+//
+using System;
+using Remotion.ServiceLocation;
+
+namespace Remotion.Web.ExecutionEngine
+{
+  /// <summary>
+  /// Contains relevant settings for the URL functionality of the web execution engine.
+  /// </summary>
+  [ImplementationFor(typeof(WxeUrlSettings), Lifetime = LifetimeKind.Singleton)]
```

Comment:
I would assume that we are missing an "IoC test" here as well even if we don't have an interface. We probably have the same setup but assert the default values as well? (see other comment on interface for more context)